subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/SubscriptionsManager.kt

```diff
@@ -359,7 +373,11 @@ class RealSubscriptionsManager @Inject constructor(
                             logcat(LogPriority.DEBUG) { "Subs: auth token expired" }
                             val subscriptionsData = recoverSubscriptionFromStore()
                             if (subscriptionsData is Success) {
-                                AuthToken.Success(authDataStore.authToken!!)
+                                return if (subscriptionsData.entitlements.isEmpty()) {
```

Comment:
This was a bug where we weren't checking entitlements after refreshing an expired token so it looked like you had a subscription for a second when you didn't.